tests/functional/services/flows/test_validate_run.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+import pytest
+
+from globus_sdk import FlowsAPIError, SpecificFlowClient
+from globus_sdk._testing import load_response
+
+
+def test_validate_run(specific_flow_client_class: type[SpecificFlowClient]):
+    metadata = load_response(SpecificFlowClient.validate_run).metadata
+
+    flow_client = specific_flow_client_class(flow_id=metadata["flow_id"])
+
+    resp = flow_client.validate_run(**metadata["request_params"])
```

Comment:
Would it be better to remove a layer of indirection here, and store `body` in `metadata` instead of the wrapper, `request_params`?
Then we'd write this line:
```suggestion
    resp = flow_client.validate_run(body=metadata["request_body"])
```

Personally, I prefer this style over expansion of a dict sourced from elsewhere in the codebase. It means that when I read the test, I know what call signature for `validate_run` is being tested. Then, if `validate_run` gains a new parameter, we'll be able to 'see" that this test doesn't use that new param.

Reply:
Works for me!